test/models/database/challenge/challenge_test.dart

```diff
@@ -0,0 +1,18 @@
+import "package:cloud_firestore/cloud_firestore.dart";
+import "package:flutter_test/flutter_test.dart";
+import "package:proxima/models/database/challenge/challenge_data.dart";
+
+void main() {
+  test("Hash and == works", () {
+    final now = Timestamp.now();
+    ChallengeData a = ChallengeData(isCompleted: false, expiresOn: now);
```

Comment:
I have two questions on this file:
- Why is named `challenge_test` and not `challenge_data_test`?
- Why is there no test for `ChallengeFirestore` `==` and `hashCode` too? (Note that this question is not applicable if you decided to merge `ChallengeFirestore` and `ChallengeData` as I proposed earlier)

Reply:
I changed the names and added a test for ChallengeFirestore. 4696efafb6b5b742a7e079bbde36ebb16b4f88e8